src/main/java/teammates/common/util/Templates.java

```diff
@@ -196,4 +196,54 @@ public static String populateTemplate(String template, String... values) {
                 FileHelper.readResourceFile("feedbackSessionTeamEvaluationTemplate.json");
     }
     
+    public static class FeedbackQuestionDetailsConsts {
+        public static final String QUESTION_INDEX = "${qnIdx}";
+        public static final String RESPONSE_INDEX = "${responseIdx}";
+        public static final String DISABLED = "${disabled}";
+        public static final String FEEDBACK_RESPONSE_TEXT = "${Const.ParamsNames.FEEDBACK_RESPONSE_TEXT}";
+        public static final String CHECKED = "${checked}";
+        public static final String OPTION_RECIPIENT_DISPLAY_NAME = "${optionRecipientDisplayName}";
+        public static final String QUESTION_NUMBER = "${questionNumber}";
+        
+        // MCQ
+        public static final String MCQ_CHOICE_VALUE = "${mcqChoiceValue}";
+        
+        // MSQ
+        public static final String MSQ_CHOICE_VALUE = "${msqChoiceValue}";
+        public static final String MSQ_CHOICE_TEXT = "${msqChoiceText}";
+        
+        // Numscale
+        public static final String MIN_SCALE = "${minScale}";
+        public static final String MAX_SCALE = "${maxScale}";
+        public static final String STEP = "${step}";
+        public static final String CONST_PARAMSNAMES_FEEDBACK_QUESTION_NUMSCALE_MIN = "${Const.ParamsNames.FEEDBACK_QUESTION_NUMSCALE_MIN}";
+        public static final String CONST_PARAMSNAMES_FEEDBACK_QUESTION_NUMSCALE_MAX = "${Const.ParamsNames.FEEDBACK_QUESTION_NUMSCALE_MAX}";
+        public static final String CONST_PARAMSNAMES_FEEDBACK_QUESTION_NUMSCALE_STEP = "${Const.ParamsNames.FEEDBACK_QUESTION_NUMSCALE_STEP}";
```

Comment:
Why sometimes the question type is given as prefix (e.g. `CONST`), but sometimes not?